app/mixins/inplace.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-export default Ember.Mixin.create({
-    i18n: Ember.inject.service(),
+const { computed, inject, Mixin } = Ember;
+const { service } = inject;
```

Comment:
It's the double destructure that I really like.  So you get `store: service()` in the component.  That's very clean.